docs/modules/ROOT/pages/how-to-guides/Secure-your-supply-chain/proc_upgrade_build_pipeline.adoc

```diff
@@ -46,4 +46,8 @@ After upgrading the build pipeline, you can also customize it:
 . Find the new `.tekton` directory, which is the result of upgrading the build pipeline. 
 . Modify the contents of this directory to change how {ProductName} builds the component. 
 
-
+[NOTE]
+====
+{ProductName} creates pull requests with updates to pipelines definitions under the `.tekton` directory.
```

Comment:
{ProductName} creates pull requests that contain updates to the pipeline definitions under the `.tekton` directory. {ProductName} recommends you allow the execution of tests on these pull requests by creating a comment containing `/ok-to-test` and then merging the changes when the tests pass.

Reply:
Updated, thanks

Reply:
As per our style guide:
![image](https://github.com/redhat-appstudio/docs.stonesoup.io/assets/90042568/2683e1c9-7a99-4068-8657-7698f7f29949)

Reply:
Updated to the updated comment ;-)